flake-registry.json

```diff
@@ -1,215 +1,5 @@
-        "id": "blender-bin",
-        "type": "indirect"
-      },
-      "to": {
-        "dir": "blender",
-        "owner": "edolstra",
-        "repo": "nix-warez",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "cachix",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "cachix",
-        "repo": "cachix",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "composable",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "ComposableFi",
-        "repo": "composable",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "disko",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "disko",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "dreampkgs",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "dreampkgs",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "dwarffs",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "edolstra",
-        "repo": "dwarffs",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "emacs-overlay",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "emacs-overlay",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "fenix",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "fenix",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "flake-parts",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "hercules-ci",
-        "repo": "flake-parts",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "flake-utils",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "numtide",
-        "repo": "flake-utils",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "gemini",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "flake-gemini",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "hercules-ci-effects",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "hercules-ci",
-        "repo": "hercules-ci-effects",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "hercules-ci-agent",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "hercules-ci",
-        "repo": "hercules-ci-agent",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "home-manager",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "home-manager",
-        "type": "github"
-      }
-    },
```

Comment:
Let's keep home-manager given how important it is in our ecosystem

Reply:
This one I could agree with.